utils/useDataFetching.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
```

Comment:
So here we can import the `MetricQuery` directly and instantiate the graphql client

```suggestion
import React from 'react'
import { MetricQuery } from '../graphql'

const client = new GraphQLClient(
  process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT_US_EAST_2
)
```

Reply:
Done!